App.js

```diff
@@ -25,6 +26,7 @@ export default function App(props) {
       </View>
     );
   }
+
```

Comment:
Very minor: would prefer no whitespace changes so we have fewer file changes in the merge

Not blocking